trimmer_decorator.rb

```diff
@@ -0,0 +1,6 @@
+require '.\decorator'
+class TrimmerDecorator < Decorator
+  def correct_name
+    super[0..9]
```

Comment:
- [ ] Please kindly check my detailed comments on the `trimmer_decorator` file to effect changes suggested by the last reviewer here which I explained very well. 👍

Reply:
I have already given an answer for this the code is giving the correct output. Anyways, I have made the changes.